.editorconfig

```diff
@@ -1,4 +1,7 @@
 # https://editorconfig.org
+#
+# SPDX-FileCopyrightText: 2021-2024 Nextcloud GmbH and Nextcloud contributors
```

Comment:
```suggestion
# SPDX-FileCopyrightText: Nextcloud contributors
```
Let's use the same everywhere

Reply:
Strictly speaking we would than need to use Nextcloud GmbH and Nextcloud contributors in case an employee has been contributing

Reply:
Android devs added it usually, i.e. https://github.com/nextcloud/android/blob/master/app/src/main/java/com/nextcloud/ui/composeActivity/ComposeActivity.kt#L4-L5 and hence also in the generic copyright line.

Reply:
> Nextcloud GmbH and Nextcloud contributors

This is also already used in other places, even already in this repository (the openapi workflow).

Reply:
So then this it is. But let's remove the year as it is not required and avoids stupid commits and updates every year.

I'd just prefer the generic line with an authors file, so we don't start adding hundreds of authors again in the source code files.

Reply:
Do you have a reference reflecting that the year part is optional?

Reply:
> Do you have a reference reflecting that the year part is optional?

https://reuse.software/faq/#standard-copyright

> Generally, we recommend that you use SPDX-FileCopyrightText: [year] [copyright holder] <[contact address]>. You may choose to drop items **except the copyright holder**, which must always be included. We recommend that you include all items, however.

Reply:
Ah, found it
https://reuse.software/spec/#format-of-copyright-notices

Year is optional but actually reflects the initial year, doesn't have to be updated.

So even with the generic, let's have at least the creation year in. This is still what any other large Corp like Google does.

Doesn't need to be updated, but is better than no year at all.

Reply:
Yeah, bumping isn't needed and script bump all files is just noise 👍

The let's just roll will the creation date 👍

Reply:
Creation data or date of first contribution (if there are multiple authors) sounds good to me too :+1: